chunky/src/java/se/llbit/math/Grid.java

```diff
@@ -213,9 +234,14 @@ public static Grid load(DataInputStream in) throws IOException {
       throw new RuntimeException("Unknown grid format version, can't load the grid");
```

Comment:
I guess it shouldn't be too hard to make this backward-compatible? If version is 1, then offsetXYZ=0 and sizeXYZ=readInt(), right?